conf/cassandra.yaml

```diff
@@ -1168,3 +1168,20 @@ back_pressure_strategy:
 # The full query log will recrusively delete the contents of this path at
 # times. Don't place links in this directory to other parts of the filesystem.
 #full_query_log_dir: /tmp/cassandrafullquerylog
+
+
+# Audit logging - Logs every incoming CQL command request, Authentication (successful as well as unsuccessful login)
+# to C* node. Currently there is only one implementation "FileAuditLogger" which logs events to audit/audit.log file
+# using C* logging impelementation (logback) Custom logger can be implemented and injected here with the class name.
+# system keyspaces are excluded from audit log.
+# enabled: This option enables/ disables audit log
+# logger: Class name of the logger/ custom logger
+# includedKeyspaces: comma separated list of keyspaces to be included for audit log, default - includes all keysapces
+# excludedKeyspaces: comma separated list of keyspaces to be excluded for audit log, default - excludes no keysapce
+# includedCategories: comma separated list of Audit Log Categories to be included for audit log, default - includes all categories
```

Comment:
These should probably be snake_case instead of camel_case.

Once the ticket/code quiets down we can write some docs in the docs tree :-)